pkg/helmrelease/utils/helmrepoutils.go

```diff
@@ -685,7 +685,11 @@ func downloadFileHTTP(parentNamespace string, configMap *corev1.ConfigMap,
 
 		klog.V(5).Info("Download chart form helmrepo succeeded: ", fileURL)
 
-		defer resp.Body.Close()
+		defer func() {
+			if err := resp.Body.Close(); err != nil {
+				klog.Error("Error closing response: %s", err)
```

Comment:
Do we need the %s as it's klog.Error not klog.Errorf?

Reply:
We do not, forgot to remove it thanks!